assets/css/src/components-form-editor/_template-selection.scss

```diff
@@ -27,11 +27,19 @@ $mailpoet-form-template-thumbnail-height: 316px;
 }
 
 .template-selection-header {
-  align-items: center;
   background: $color-input-background;
   border-bottom: 1px solid $color-tertiary-light;
-  display: flex;
-  flex-wrap: wrap;
-  justify-content: center;
+  display: grid;
```

Comment:
could you please explain why did you change this to a grid? I can see only a single line in there...

Reply:
I added the button so there are 2 items in the top bar. I needed heading to be positioned in centre and button on right. I found out that it is quite tricky to do with flexbox and I found using the grid as the most elegant solution.

Reply:
I reckon that grid "trick" deserves a comment in the code, what do you think?